Fix numpy data types and use ICPT binary from Dice in EasyBuild
author: Alex Domingo (Vrije Universiteit Brussel)
--- pyscf/icmpspt/icmpspt.py.orig	2022-11-25 01:00:16.277514709 +0100
+++ pyscf/icmpspt/icmpspt.py	2022-11-24 21:38:23.700229771 +0100
@@ -43,7 +43,7 @@
 from pyscf import tools
 import sys
 
-float_precision = numpy.dtype('Float64')
+float_precision = numpy.dtype('float64')
 mpiprefix=""
 executable="/projects/bamu3429/softwares/icpt_test/icpt"
 executable="/projects/bamu3429/softwares/icpt/icpt.big"
--- pyscf/icmpspt/icmpspt.py.orig	2022-11-25 01:00:16.277514709 +0100
+++ pyscf/icmpspt/icmpspt.py	2022-11-25 01:03:20.021383507 +0100
@@ -45,10 +45,7 @@
 
 float_precision = numpy.dtype('Float64')
 mpiprefix=""
-executable="/projects/bamu3429/softwares/icpt_test/icpt"
-executable="/projects/bamu3429/softwares/icpt/icpt.big"
-executable="/projects/bamu3429/softwares/icpt/icpt"
-#executable="/projects/bamu3429/softwares/icpt_before_me/icpt"
+executable=os.path.join(os.getenv('EBROOTDICE'), 'bin', 'ICPT')
 if not os.path.isfile(executable):
     msg = ('MPSLCC executable %s not found.  Please specify "executable" in %s'
            % (executable, __file__))
